cypress/test/footer/footer.cy.js

```diff
@@ -0,0 +1,39 @@
+import { mount } from '@cypress/react'
+import Footer from '../../../components/footer/Footer'
+import { socialMediaLinks } from '../../../components/footer/FooterList'
+import { initiativeLinks } from '../../../components/footer/FooterList2'
+
+describe('Footer component', () => {
+  beforeEach(() => {
+    mount(<Footer />)
+  })
+
```

Comment:
Add a case for checking AsyncAPI logo in footer and the link connected to the logo.